### ISSUE TITLE
Which weights to load 

### ISSUE THREAD

**Issue by wenxin-bupt** [anuragranj/cc#12]
Hi,  I have two questions about the training details.

1. Which weights do you use when starting a new cc training phase: the "best" weights selected through validation during the training of last phase    **or**    the weights saved in the end of last phase.

2.  When training depth&pose in competition step,  weights λ is set to (1.0, 0.5, 0.05, 0). Since flownet and  masknet are fixed, why set the  λ flowloss to 0.5 and λ exploss 0.05? (It doesn't make much difference anyway.)

Thank you.

**Comment by anuragranj** [comment]
1. We tried both, but it doesn't matter much. The error just saturates (doesn't increase). For the reported results, we did use the best model from the validation set.
2. λ_depth = 1.0, λ_flow = 0.5. This is because depth uses 4 frame pairs on the loss (I--, I-, I+, I++) paired with I, and flow uses 2 frame pairs (I-. I) paired with I. This is used while training masknet. For training flownet and depth-pose net, it is okay to keep the corresponding λs to 1, and λs for other networks do not effect the training of network being trained. However, we made this choice keeping the λs consistent across training different networks, because we wanted to keep things simple by not changing λs too many times.
